ci/mac_ci_steps.sh

```diff
@@ -32,6 +32,7 @@ DEFAULT_TEST_TARGETS=(
   "//test/integration:protocol_integration_test"
   "//test/integration:tcp_proxy_integration_test"
   "//test/integration:extension_discovery_integration_test"
+  "//test/extensions/dynamic_modules/http:filter_test"
```

Comment:
note: this is the minimal test that ensures the dynamic loading/symbol resolution working